nixos/tests/nominatim.nix

```diff
@@ -0,0 +1,187 @@
+{ pkgs, lib, ... }:
+
+let
+  # Andorra - the smallest dataset in Europe (3.1 MB)
+  osmData = pkgs.fetchurl {
+    url = "https://web.archive.org/web/20250430211212/https://download.geofabrik.de/europe/andorra-latest.osm.pbf";
+    hash = "sha256-Ey+ipTOFUm80rxBteirPW5N4KxmUsg/pCE58E/2rcyE=";
+  };
+in
+{
+  name = "nominatim";
+  meta = {
+    maintainers = with lib.teams; [
+      geospatial
+      ngi
+    ];
+  };
+
+  nodes = {
+    # nominatim - self contained host
+    nominatim =
+      { config, pkgs, ... }:
+      {
+        # Nominatim
+        services.nominatim = {
```

Comment:
Is it possible to access the service from the host machine? I tried forwarding the ports, but it refuses my connection:

```nix
virtualisation.forwardPorts = [
  {
    host.port = 9090;
    guest.port = 80;
  }
];
```

```
nominatim # [  121.275089] refused connection: IN=eth0 OUT= MAC=52:54:00:12:34:56:52:55:0a:00:02:02:08:00 SRC=10.0.2.2 DST=10.0.2.15 LEN=44 TOS=0x00 PREC=0x00 TTL=64 ID=18 PROTO=TCP SPT=57504 DPT=80 WINDOW=65535 RES=0x00 SYN URGP=0
```